### PR TITLE
Add `created` and `updated` times to membership API responses

### PR DESCRIPTION
**Pull request by seanh**
Fixes https://github.com/hypothesis/h/issues/9137

### CODE REVIEW

File: h/presenters/group_membership_json.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
     EditGroupMembershipContext,
     GroupMembershipContext,
 )
+from h.util.datetime import utc_iso8601
```
Comment (marcospri) [inline h/presenters/group_membership_json.py:7]: This handles None, no problem: 

https://github.com/hypothesis/h/blob/main/h/util/datetime.py#L8

File: tests/unit/h/presenters/group_membership_json_test.py [hunk 2]
```diff
@@ -39,8 +43,19 @@ def test_it_with_permissive_securitypolicy(
                 "updates.roles.admin",
                 "updates.roles.owner",
             ],
+            "created": "1970-01-01T00:00:00.000000+00:00",
+            "updated": "1970-01-01T00:00:01.000000+00:00",
         }
 
+    def test_it_with_no_created_or_updated_times(self, membership, pyramid_request):
```
Comment (marcospri) [inline tests/unit/h/presenters/group_membership_json_test.py:50]: I don't think we need a test for it but there will be cases where created will stay None but updated will have a value.

**Review by marcospri — APPROVED** [review]
